deploy/determined_deploy/aws/aws.py

```diff
@@ -14,10 +14,12 @@
 def get_user(boto3_session: boto3.session.Session) -> str:
     sts = boto3_session.client("sts")
     response = sts.get_caller_identity()
-    return response["Arn"].split("/")[-1]
+    user = response["Arn"].split("/")[-1]
+    assert isinstance(user, str), f"expected a string representing the user but got {user}"
```

Comment:
non-blocking: I feel like it would make more sense to have the assertion on `response["Arn"]` directly and then do string operations from there.

Reply:
Yeah that makes sense.